jsonnet/ibm-cloud-managed-profile.libsonnet

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Can we add a comment saying the intent + a point of contact?

Reply:
done, thx!